docs/presto-admin-commands.rst

```diff
@@ -489,14 +489,14 @@ server upgrade
 **************
 ::
 
-    presto-admin server upgrade local_package_path [local_config_dir]
+    presto-admin server upgrade new-rpm [local_config_dir]
```

Comment:
What if you put `path/to/new/package.rpm`. Isn't it clear that you have to provide path with filename?